src/lib/OpenEXRCore/internal_zip.c

```diff
@@ -24,8 +24,8 @@
 #    define IMF_HAVE_SSE4_1 1
 #    include <smmintrin.h>
 #endif
-#if defined(__ARM_NEON)
-#    define IMF_HAVE_NEON 1
+#if defined(__aarch64__)
+#    define IMF_HAVE_NEON_AARCH64 1
```

Comment:
Doesn't this duplicate what's in `ImfSimd.h` above?

Reply:
Yes, but internal_zip.c already did this for all SIMD sets including SSE. I don't know the reasoning, just went with the flow.

Reply:
Right, this file doesn't include ImfSimd.h, so that's appropriate.